paper/paper.md

```diff
@@ -0,0 +1,519 @@
+    index: 11
+date: 30 Apr. 2021
+bibliography: paper.bib
+---
+
+<!-- TODO: add suggestions from Maureen: I would avoid using contractions (e.g. “couldn’t” in the Summary), and it may be helpful to readers if you could provide some examples of the applications students used for their projects and what they usually learned from the projects (could be things other than technical skills). -->
+
+# Summary
+
+<!-- pasted from our PyCon 2020 talk proposal -->
+We are bioinformatics trainees at the University of Michigan who started a
+local chapter of Girls Who Code to provide a fun and supportive environment for
+high school women to learn the power of coding.
+Our goal was to cover basic coding topics and data science concepts through
+live-coding and hands-on practice.
+However, we couldn’t find a resource that exactly met our needs. Therefore, over the past
+three years, we have developed a curriculum and instructional format using
+Jupyter notebooks to effectively teach introductory Python for data science.
+This method, inspired by The Carpentries organization, uses bite-sized lessons
+followed by independent practice time to reinforce coding concepts, and
+culminates in a data science capstone project using real-world data.
+We believe our open curriculum is a valuable resource to the wider education
+community and hope that educators will use and improve our lessons, practice
+problems, and teaching best practices.
+Anyone can contribute to [our educational materials on GitHub](https://github.com/GWC-DCMB).
+
+
+# Statement of Need
+
+As women bioinformatics trainees at the
+University of Michigan, we experience the gender gap in our field first-hand.
+During the 1974-1975 academic year, women achieved 18.9% of total Bachelor's
+degrees in computer and information sciences in the US [@nces_digest_2012].
+By 1983-1984 this peaked at 37.1%, but fell to 17.6% by 2010-2011. We also see this
+national trend in the training of the next generation of Bioinformaticians at
+Michigan Medicine.
+Since accepting its first students in 2001, the U-M Bioinformatics Graduate
+Program has graduated 66 male and 22 female doctorates as of 2019.
+This disparity begins at the applicant level; during 2016-2019 the average
+percentage of females applying directly to the Bioinformatics PhD program was
+35.2%, and the average percentage of female applicants listing Bioinformatics as first or second
+choice in the Program in Biomedical Sciences, U-M's biomedical PhD umbrella
+program was 41%.
+
+Previous research on women's educational experiences in science, technology, engineering,
+and mathematics (STEM) have produced
+various explanations for persistent gender disparities
+[@benbow_gender_2016].
+One explanation is that women in first year computer science courses rate
+themselves as being less experienced, prepared, or able to master the course
+material than the men [@fisher_undergraduate_1997].
```

Comment:
I think we should make clearer why this is an explanation. e.g. does this perception directly lead to poorer performance? Alternatively, we could delete this sentence and just include the next one as the example. @bnwolford thoughts?

Reply:
I deleted it, IMO less is often more 605d8a6